tsconfig.json

```diff
@@ -9,7 +9,7 @@
     "noEmit": true,
     "esModuleInterop": true,
     "module": "esnext",
-    "moduleResolution": "node",
+    "moduleResolution": "Bundler", // allow following "exports" in package.json. This is also fine as the code isn't compiled by `tsc` but by Next / Astro
```

Comment:
This is required otherwise next compilation is failing with this error:

<img width="1031" alt="image" src="https://github.com/user-attachments/assets/54550368-498c-47b9-b921-d02de71a1598">

(`astro/errors` exists but is defined in the `exports` field in `astro`'s package.json)

Reply:
thanks. this is really helpful!